repldex/backend/website.py

```diff
@@ -520,6 +520,8 @@ async def middleware(request, handler):
 	request.is_admin = is_admin
 	request.discord_id = discord_id
 	resp = await handler(request)
+	if resp.status in [404, 500, 418]:
+		return web.HTTPFound('/entry/'+str(resp.status))
```

Comment:
there should be spaces around the plus sign

Reply:
done